Framework/DataHandling/src/LoadIsawDetCal.cpp

```diff
@@ -210,15 +210,16 @@ void LoadIsawDetCal::exec() {
       }
     }
     boost::shared_ptr<RectangularDetector> det;
-    std::ostringstream Detbank;
-    Detbank << "bank" << id;
+    std::string Detbank = "bank" + std::to_string(id);
```

Comment:
This has nothing to do with these changes, but a bit further up (L135), there is an exception for WISH, where my understanding from the code is that `bankName` is not `bank` but `WISHpanel`. I'm not sure this is a bug, but it might be.